packages/core/config/jest.config.web.js

```diff
@@ -1,6 +1,12 @@
 // used by cli `rw test` command
-// note: rootDir is a workaround for jest working directory weirdness
+
+const path = require('path')
+
 module.exports = {
   resolver: 'jest-directory-named-resolver',
   rootDir: process.cwd(),
+  globals: {
```

Comment:
Super curious to learn about what you had to do here… wowza.

Reply:
Oh, very interesting —> https://jestjs.io/docs/en/configuration#globals-object

FYI: we had someone asking about how we handle Globals. Might be a good future article in the forum, fwiw.